mk/uvmt/uvmt.mk

```diff
@@ -331,11 +331,15 @@ isacov_logdiff:
 		@ls $(ISACOV_TRACELOG) > /dev/null
 		@ls $(ISACOV_AGENTLOG) > /dev/null
 	@echo filtering logs...
-		@cat $(ISACOV_TRACELOG) \
-			| sed 's/\(.*\)   \(.*\)/\1/' | awk '{$$1=$$2=$$3=$$4=$$5=""; $$0=$$0; $$1=$$1; print $$0}' \
-			| tail -n +2 > trace.tmp
+		@cat $(ISACOV_TRACELOG)              \
```

Comment:
Can we have a quick explanation of what the awk/sed-magic here does?

Reply:
Fixed.